src/Formulate.js

```diff
@@ -7,6 +7,7 @@ import isPlainObject from 'is-plain-object'
 import { en } from '@braid/vue-formulate-i18n'
 import fauxUploader from './libs/faux-uploader'
 import FormulateSlot from './FormulateSlot'
+import FormulateSchema from './FormulateSchema.vue'
```

Comment:
@justin-schroeder reason for the .vue in the import here?

Reply:
Also, this is the programatic submit PR, the Schema stuff should not be in here, no?

Reply:
yeah the rollup bundler chokes without it